index.html

```diff
@@ -1,42 +1,88 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <title>Spotify</title>
+    <meta charset="UTF-8">
+    <title>Spotify</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
+
+
+    <link rel="stylesheet" href="/lab-css-spotify-clone/css/main.css">
+
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+    <header>
+        <img src="/lab-css-spotify-clone/images/spotify-logo.png" alt="logo">
+        <ul>
+            <a href="">
+                <li>Premium</li>
+            </a>
+            <a href="">
+                <li>Discover</li>
+            </a>
+            <a href="">
+                <li>Help</li>
+            </a>
+            <a href="">
+                <li>Download</li>
+            </a>
+        </ul>
+
+
+    </header>
+
+    <section class=top>
+        <h2>Music for everyone.</h2>
+        <h4>Spotify is now free on mobile, tablet and computer. <br> Listen to the right music, wherever you are.</h4>
+    </section>
+
+    <main>
+        <section class="middle">
+
+            <h4>What’s on Spotify? </h4>
+            <section>
+                <div>
+                    <img src="/lab-css-spotify-clone/images/music-icon.png" alt="a">
+                    <h4>Millions of Songs</h4> There are millions of songs on Spotify </div>
+                <div>
+                    <img src="/lab-css-spotify-clone/images/high-quality-icon.png" alt="b">
+                    <h4>HD Music</h4> Listen to music as if you were listening live</div>
+                <div>
+                    <img src="/lab-css-spotify-clone/images/devices-icon.png" alt="c">
+                    <h4>Stream Everywhere</h4> Stream music on your smartphone, tablet or computer</div>
+            </section>
+
+        </section>
+
+        <section class="bottom">
+            <div class=left>
+                <h4>It’s as yeezy as Kanye West.</h4>
+
+
+                <div class="article">
+                    <h3>Search</h3> Know what you want to listen to? <br> Just search and hit play.</div>
```

Comment:
The text after the h3 should be inside a `<p>` tag
Having a `<br>` tag break the paragraph in lines is not a very good practice. The size of the container should take care of that instead.